go.mod

```diff
@@ -414,6 +414,8 @@ require (
 	sigs.k8s.io/yaml v1.2.0 // indirect
 )
 
+require github.com/DataDog/btf-internals v0.0.0-20220317155923-e1a7b770b6a1
```

Comment:
This should be in the top-level `require` block, correct?

Reply:
Yes, I'm not sure what `inv -e tidy-all` did here